plutus-pab/app/Cli.hs

```diff
@@ -200,6 +199,8 @@ runCliCommand _ _ _ _ (ContractState uuid) =
 runCliCommand _ _ _ _ ReportInstalledContracts = do
     logInfo InstalledContractsMsg
     traverse_ (logInfo . InstalledContract . render . pretty) =<< Core.installedContracts @ContractExe
+        where
+            render = renderStrict . layoutPretty defaultLayoutOptions
```

Comment:
This was the only place where `render` from the now removed `Utils` module was used. I dropped it and used this local binding instead.